### DISCUSSION TITLE
希望能增加一些伺服器的常用操作指令

### DISCUSSION THREAD

**Discussion by FishYu-OWO** [lollipopkit/flutter_server_box#119] [Ideas]
是否能增加一些按鈕，直接控制Server關機，重啟等功能
雖然這些功能用簡單的指令就能做到，但是畢竟手機下指令有點不方便
是否能增加一些簡單的快捷功能

**Comment by lollipopkit** [comment]
- 已在 v521 中实现
- 长按会显示如下菜单
![Screenshot 2023-09-01 at 12 32 37](https://github.com/lollipopkit/flutter_server_box/assets/10864310/5f973d6e-1062-485a-a1ec-c9fa4b1e665e)
